.circleci/config.yml

```diff
@@ -141,22 +141,20 @@ workflows:
           requires:
             - request-preprod-approval
 
-  #      - request-prod-approval:
-  #          type: approval
-  #          requires:
-  #            - deploy_preprod
-  #      - hmpps/deploy_env:
-  #          name: deploy_prod
-  #          env: "prod"
-  #          jira_update: true
-  #          jira_env_type: production
-  #          slack_notification: true
-  #          slack_channel_name: << pipeline.parameters.releases-slack-channel >>
-  #          context:
-  #            - hmpps-common-vars
-  #            - approved-premises-ui-prod
-  #          requires:
-  #            - request-prod-approval
+      - request-prod-approval:
+          type: approval
+          requires:
+            - deploy_preprod
+      - hmpps/deploy_env:
+          name: deploy_prod
+          env: "prod"
+          slack_notification: true
+          slack_channel_name: << pipeline.parameters.releases-slack-channel >>
```

Comment:
Is it worth backfilling these slack setting to pre-prod to verify that we've got the wiring and permissions working? (Assuming we do feel that slack notifications for pre-prod as well as prod are of value...)

Reply:
I note that MRD have configured a slack action for build failure: https://github.com/ministryofjustice/make-recall-decision-ui/blob/25b3e73462e577516a79c6014a9a113033b8d842/.circleci/config.yml#L33-L61

might be worth copying, or making a card in the backlog to copy...

Reply:
Yes it's true we haven't tested that slack notification. I was thinking that because we have [CI job failure events coming through on Slack](https://mojdt.slack.com/archives/C048BJS7S2F/p1675234943024679) we have enough of a signal for non production environments. For production I was thinking that we want something even more shouty and that might alert the wider team of a potential issue. 

I'll add a link to what MRD have done in case we feel the need to reach for it.

Reply:
Okay it looks like that slack event is coming from this line https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/blob/f8e442ad46eb65de49048317cbd62d20d2a6f1f3/.circleci/config.yml#L41 and a few others.

I think we ship this to get production notifications going the same way and sweep back to add them for prod. I'll put something in our backlog.